doc/Overview/Mvux/Advanced/Selection.md

```diff
@@ -0,0 +1,227 @@
+        </Grid.RowDefinitions>
+
+        <StackPanel DataContext="{Binding SelectedPerson}" Orientation="Horizontal" Spacing="5">
+            <TextBlock Text="Selected person:" />
+            <TextBlock Text="{Binding FirstName}"/>
+            <TextBlock Text="{Binding LastName}"/>
+        </StackPanel>
+
+        <Border Height="2" Background="Gray" Grid.Row="1" />
+
+        <ListView Grid.Row="2" ItemsSource="{Binding People}">
+    </Grid>
+</Page>
+```
+
+When running the app, the top section will reflect the item the user selects in the `ListView`:
+
+![](../Assets/Selection.gif)
+
+> [!NOTE]  
+> The source-code for the sample app can be found [here](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MvuxHowTos/AdvancedPeopleApp).
+
+### Listening to the selected value
+
+You can listen and detect selection changes by either creating a feed that projects the selection with the `Select` operator, or by subscribing to the selection feed using the `ForEach` operator.
+
+#### Using the Select operator
+
+Using the example above, we can project the `SelectedPerson` property to project or transform the current `Person`, using the `SelectedPerson``s [`Where`](xref:Overview.Mvux.Feeds#where) and [`Select`](xref:Overview.Mvux.Feeds#select) operators.
+
+```csharp
+public IFeed<string> GreetingSelect => SelectedPerson.Select(person => person == null ? string.Empty : $"Hello {person.FirstName} {person.LastName}!");
+```
+
+A `TextBlock` can then be added in the UI to display the selected value:
+
+```xml
+<TextBlock Text="{Binding GreetingSelect}"/>
+```
+
+#### Using the ForEach operator
+Selection can also be propagated manually to a State using the [`ForEach`](xref:Overview.Mvux.Feeds#foreach) operator.
+First we need to create a State with a default value, which will be used to store the processed value once a selection has occurred.
+
+```csharp
+public IState<string> GreetingForEach => State.Value(this, () => string.Empty);
+```
+
+In the constructor we can then subscribe to the value in the following manner:
+
+```csharp
+public partial record PeopleModel
+{
+    private IPeopleService _peopleService;
+
+    public PeopleModel(IPeopleService peopleService)        
+    {
+        _peopleService = peopleService;
+
+        SelectedPerson.ForEachAsync(action: SelectionChanged);
+    }
+
+    ...
+
+    public async ValueTask SelectionChanged(Person? selectedPerson, CancellationToken ct)
+    {
+        if (selectedPerson == null)
+            return;
+
+        await GreetingForEach.Set($"Hello {selectedPerson.FirstName} {selectedPerson.LastName}!", ct);
+    }
+}
+```
+
+The `ForEach` operator listens to an selection occurrence and invokes the `SelectionChanged` callback with the newly available data, in this case the recently selected `Person` entity.
+
+> [!TIP]  
+> MVUX takes care of the lifetime of the subscription, so it will be disposed of along with its declaring `Model` being garbage-collected.
```

Comment:
```suggestion
The `ForEach` operator listens to a selection occurrence and invokes the `SelectionChanged` callback with the newly available data, in this case, the recently selected `Person` entity.

> [!TIP]  
> MVUX takes care of the lifetime of the subscription, so it will be disposed of along with its declaring `Model` being garbage-collected.
```